api/capi/src/nnstreamer-capi-single.c

```diff
@@ -51,7 +51,9 @@ typedef struct
   ml_tensors_info_s in_info;
   ml_tensors_info_s out_info;
 
-  guint timeout; /* milliseconds */
+  GMutex lock; /**< Lock for internal values */
+  gboolean timed_out; /**< Previous buffer was timed out */
```

Comment:
It seems that timed_out and timeout are similar too. Could you improve readability for maintenance?

Reply:
will be changed to 'clear_previous_buffer'